app/services/idv/steps/link_sent_step.rb

```diff
@@ -1,7 +1,48 @@
 module Idv
   module Steps
     class LinkSentStep < DocAuthBaseStep
-      def call; end
+      def call
+        dac = DocCapture.find_by(user_id: current_user.id)
+        flow_session[:instance_id] = dac.acuant_token
+
+        failure_data, data = verify_back_image
+        return failure_data if failure_data
+
+        extract_pii_from_doc(data)
+
+        mark_steps_complete
+      end
+
+      private
+
+      def mark_steps_complete
+        %i[send_link link_sent email_sent mobile_front_image mobile_back_image front_image
+           back_image].each do |step|
+          mark_step_complete(step)
+        end
+      end
+
+      def extract_pii_from_doc(data)
+        pii_from_doc = Idv::Utils::PiiFromDoc.new(data).call(
+          current_user.phone_configurations.first.phone,
+        )
+        flow_session[:pii_from_doc] = pii_from_doc
+      end
+
+      def verify_back_image
+        back_image_verified, data = assure_id.results
+        return failure(data) unless back_image_verified
+
+        return [nil, data] if data['Result'] == 1
+
+        failure_alerts(data)
+      end
+
+      def failure_alerts(data)
+        failure(data['Alerts'].
+          reject { |res| res['Result'] == 2 }.
```

Comment:
Maybe something like this would be easier to read:
```ruby
    BAD_RESULT = 2
    ...
          reject { |res| res['Result'] == BAD_RESULT }.
```

for some reasonably named constant that documents the result we're rejecting.